packages/expo-modules-core/android/build.gradle

```diff
@@ -97,9 +97,7 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
   implementation "org.jetbrains.kotlin:kotlin-reflect:${getKotlinVersion()}"
   implementation 'androidx.annotation:annotation:1.2.0'
-
-  // used only in `expo.modules.core.errors.ModuleDestroyedException` for API export
-  compileOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3")
+  implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3"
```

Comment:
Now I noticed that - maybe check which `kotlinx-coroutines-core` version is the latest working with current Kotlin. For example `expo-clipboard` has `1.5.1`, but I suspect there's a 1.6.0 or sth like that